cdap-standalone/src/main/resources/cdap-site.xml

```diff
@@ -164,6 +163,15 @@
     </description>
   </property>
 
+  <property>
+    <name>preview.enabled</name>
+    <value>true</value>
```

Comment:
Do we want to enable it by default? I thought we don't want it enabled by default. @sreevatsanraman ?